src/localizer.cpp

```diff
@@ -185,15 +195,29 @@ QVariant Localizer::data(const QModelIndex& index, int role) const {
 
   switch (role) {
     case LanguageRole:
-      return QVariant(languageName(m_languages.at(index.row())));
+      return QVariant(m_languages.at(index.row()).m_name);
 
     case LocalizedLanguageRole:
-      return QVariant(localizedLanguageName(m_languages.at(index.row())));
+      return QVariant(m_languages.at(index.row()).m_localizedName);
 
     case CodeRole:
-      return QVariant(m_languages.at(index.row()));
+      return QVariant(m_languages.at(index.row()).m_code);
 
     default:
       return QVariant();
   }
 }
+
+QStringList Localizer::languages() const {
+  QStringList languages;
+  for (const Language& language : m_languages) {
+    languages.append(language.m_code);
+  }
+
+  return languages;
+}
+
+bool Localizer::languageSort(const Localizer::Language& a,
+                             const Localizer::Language& b) {
+  return a.m_localizedName < b.m_localizedName;
```

Comment:
I would like to use QCollator but it doesn't work on Linux as it should.
If this doesn't work, I can investigate why we have a crash when using QCollator